packages/marko/docs/marko-vs-react.md

```diff
@@ -617,20 +617,22 @@ style.less {
 
 ### API
 
-Marko compiles component to JavaScript modules that export an API for rendering
-the component as shown below:
+Marko compiles components to JavaScript modules with exported API for rendering,
```

Comment:
I don't think this is grammatically correct

Reply:
How about the new version?